pmdarima/arima/arima.py

```diff
@@ -779,35 +810,35 @@ def update(self, y, exogenous=None, maxiter=None, **kwargs):
         y = check_endog(y, dtype=DTYPE)
         n_samples = y.shape[0]
 
-        # if exogenous is None and new exog provided, or vice versa, raise
-        exogenous = self._check_exog(exogenous)  # type: np.ndarray
+        # if X is None and new X provided, or vice versa, raise
+        X = self._check_exog(X)  # type: np.ndarray
 
         # ensure the k_exog matches
-        if exogenous is not None:
+        if X is not None:
             k_exog = model_res.model.k_exog
-            n_exog, exog_dim = exogenous.shape
+            n_exog, exog_dim = X.shape
 
-            if exogenous.shape[1] != k_exog:
-                raise ValueError("Dim mismatch in fit exogenous (%i) and new "
-                                 "exogenous (%i)" % (k_exog, exog_dim))
+            if X.shape[1] != k_exog:
+                raise ValueError("Dim mismatch in fit X (%i) and new "
+                                 "X (%i)" % (k_exog, exog_dim))
```

Comment:
Could use an f-string

```suggestion
                raise ValueError(f"Dim mismatch in fit X ({k_exog}) and new X ({exog_dim})"
```